src/devices/Button/ButtonBase.cs

```diff
@@ -104,6 +113,12 @@ protected void HandleButtonPressed()
         /// </summary>
         protected void HandleButtonReleased()
         {
+            if (_debounceTime.Ticks > 0 && !IsPressed)
```

Comment:
can you explain `!IsPressed`? Is that correct? While ignoring you should ignore both pressed and released state

Reply:
Yes because in the Pressed handler, I ignore the events coming from the driver when debouncing is occurring. This means you also get multiple Released handlers called and I need to "pair" them, otherwise the Pressed event gets called.